Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -461,6 +468,13 @@
       reagents:
       - ReagentId: SpaceCleaner
         Quantity: 30
+  - type: CartridgeAmmo #Sunrise-Start
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)